### ISSUE TITLE
Included Mermaid CSS produces some unlegible text for non-light themes

### ISSUE THREAD

**Issue by okwolf** [mjbvz/vscode-markdown-mermaid#3]
Using this sample Markdown:
````
### Sequence diagram

 ```mermaid
sequenceDiagram
loop every day
    Alice->>John: Hello John, how are you?
    John-->>Alice: Great!
end
```
````

Produces this preview using the Light+ theme in VSCode:
![screen shot 2017-08-12 at 20 39 09](https://user-images.githubusercontent.com/3735164/29245988-cf39abb6-7f9f-11e7-902b-28eb7f521936.png)

Unfortunately it also produces this output using the default Dark+ theme:
![screen shot 2017-08-12 at 20 39 29](https://user-images.githubusercontent.com/3735164/29245995-e5cdcbdc-7f9f-11e7-92b0-f5a67f720bbb.png)

I noticed you are including `./node_modules/mermaid/dist/mermaid.css` which works well for diagrams against a light theme, however there are also `mermaid.dark.css`, `mermaid.forest.css`, and `mermaid.neutral.css` available. Is there any way to get the extension to include a different style sheet based on your theme? There seems to be a comment to this effect [here](https://github.com/Microsoft/vscode/issues/22916#issuecomment-293955221) however I could not find any documentation [here](https://code.visualstudio.com/docs/languages/markdown#_using-your-own-css) or [here](https://github.com/Microsoft/vscode-docs/blob/vnext/docs/extensionAPI/api-markdown.md) confirming this.

I really like using mermaid syntax within my Markdown, but I also really like using dark themes in my editor. Currently I'm using [this](https://marketplace.visualstudio.com/items?itemName=vstirbu.vscode-mermaid-preview) extension which handles using different CSS based on your theme, but it only shows a preview of a single mermaid diagram and doesn't integrate with the rest of your Markdown preview.

**Comment by mjbvz** [comment]
This should be fixed in 0.0.3 which was just published:

![screen shot 2017-08-19 at 2 03 30 pm](https://user-images.githubusercontent.com/12821956/29490107-5dbc9910-84e7-11e7-87a0-84fce46f68ca.png)

![screen shot 2017-08-19 at 2 05 13 pm](https://user-images.githubusercontent.com/12821956/29490110-6ecb9e4a-84e7-11e7-8437-bb3e01c85978.png)

**Comment by 010101110100111101010111** [comment]
Can this issue be reopened? I've found that using dark themes this example graph renders renders as it previously did, which is very difficult to read.

Here is a screenshot using the default Dark+ theme. 

![image](https://user-images.githubusercontent.com/49313514/55633656-c574b280-57b4-11e9-92c0-141e36cc86f3.png)

It'd be great if this can be fixed given how useful this extension is.
